### PR TITLE
#24 Add Hilt

### PR DESCRIPTION
**Pull request by BorysChajdas**

### CODE REVIEW

File: app/src/main/java/com/example/android/picover/presentation/navigation/PicoverNavHost.kt [hunk 1]
```diff
@@ -24,10 +25,11 @@ fun PicoverNavHost(
 			HomeScreen()
 		}
 		composable("camera") {
-			CameraScreen()
+			CameraScreen(hiltViewModel())
```
Comment (michal-kucznerowicz) [inline app/src/main/java/com/example/android/picover/presentation/navigation/PicoverNavHost.kt]: Minor: You could pass `hiltViewModel()` as the default argument in `CameraScreen`.

**Review by michal-kucznerowicz — APPROVED** [review]

File: app/build.gradle.kts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 plugins {
```
Comment (marcinWisnia) [inline app/build.gradle.kts:1]: @BorysChajdas  please remember that now we have "kotlin" name of packages, i see that PicoverApplication and CameraViewModel are in "java" named directory.

Comment (BorysChajdas) [reply app/build.gradle.kts:1]: Yeah, rebase problem, thanks for catching it!

Comment (BorysChajdas) [reply app/build.gradle.kts:1]: Done

**Review by marcinWisnia — APPROVED** [review]

**Pull request merged**